package.json

```diff
@@ -6,7 +6,7 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test --browsers=ChromeHeadless --watch=false --code-coverage"
```

Comment:
You can have different command for test with coverage. Its standard practice to run only tests in test command.

You can add lint check here.